public/components/SponsorshipEdit/SponsorLogo.react.js

```diff
@@ -24,12 +24,19 @@ export default class SponsorEdit extends React.Component {
     this.props.onImageUpdated(undefined);
   }
 
+  constructErrorMessage(error){
+    if (error.status === 400) { // 400 errors are returned by our application, with useful responseText
+      return {errorMessage: error.responseText}
+    } else return {errorMessage: error.statusText} // Other errors are returned by nginx, where statusText is more useful
```

Comment:
best not mix with and without curlies...
```suggestion
    } else {
         return {errorMessage: error.statusText} // Other errors are returned by nginx, where statusText is more useful
    }
```